pro/src/components/SideNavLinks/SideNavLinks.tsx

```diff
@@ -186,10 +185,10 @@ export const SideNavLinks = ({ isLateralPanelOpen }: SideNavLinksProps) => {
                   </span>
                 </NavLink>
               </li>
-              {selectedOfferer?.hasPartnerPage && venueId && (
+              {selectedOfferer?.hasPartnerPage && firstHasPartnerPageVenueId && (
```

Comment:
On a encore besoin du premier check ?
```suggestion
              {firstHasPartnerPageVenueId && (
```

Reply:
Non, en effet.